### DISCUSSION TITLE
Docker compose file with ftp enabled and mysql database

### DISCUSSION THREAD

**Discussion by rovervt** [drakkan/sftpgo#1160] [General]
Hello, can anyone help me to create a docker compose file with FTP,Webdav and mysql enabled? Thanks

**Comment by JonBendtsen** [comment]
If you can not do that yourself, should you even be running software?

**Comment by rovervt** [comment]
Finally I made it, but thanks for your not help

**Reply by drakkan** [reply]
> Finally I made it, but thanks for your not help

sorry if we seemed rude, but please understand that this isn't a free help center, check the [support policy](https://github.com/drakkan/sftpgo#support-policy). Thanks. If users just take and don't return anything back, the project will die in the long run and you will be forced to pay for a similar proprietary solution.
